Docs: EC2 Section Refactoring (#163)
C2Devel/terraform-provider-croccloud: docs/data-sources/instances.md

Reviewer: 1:1
здесь вместо первого and не or д.б.?
и зпт нужна перед ним?

Reviewer: а нужен ли этот note? у нас же нет эфемерных инстансов

> instances (e.g., managed via autoscaling group), as the output may change at any time

Reviewer: ```suggestion
instances (e.g., managed via an Auto Scaling group), as the output may change at any time
```

}
```

## Argument reference

* `filter` - (Optional, [Block](#filter)) One or more name/value pairs to use as filters.
    * _Valid values:_ See supported names and values in [EC2 API documentation][describe-instances]
* `instance_state_names` - (Optional, Set of strings) List of instance states that should be applicable to the desired instances.
    * _Valid values:_ `pending`, `running`, `shutting-down`, `stopped`, `stopping`, `terminated`
* `instance_tags` - (Optional, Map of strings) Key-value pairs. Must exactly match pairs on the desired resources.

### filter

* `name` - (Required, String) The name of the filter.
    * _Constraints:_ Filter names are case-sensitive
* `values` - (Required, List of strings) One or more filter values.
    * _Constraints:_ Filter values are case-sensitive

## Attribute reference

In addition to all arguments above, the following attributes are exported:

* `id` - (String) The region.

> * `ids` - (List of strings) IDs of instances found through the filter.

Reviewer: здесь наверно определенный артикль (дается описание, что это за инстансы
```suggestion
* `ids` - (List of strings) The IDs of instances found through the filter.
```

> * `private_ips` - (List of strings) Private IP addresses of instances found through the filter.

Reviewer: ```suggestion
* `private_ips` - (List of strings) The private IP addresses of instances found through the filter.
```

> * `public_ips` - (List of strings) Public IP addresses of instances found through the filter.

Reviewer: ```suggestion
* `public_ips` - (List of strings) The public IP addresses of instances found through the filter.
```
